build/src/transformTs.js

```diff
@@ -0,0 +1,186 @@
+const wr = require('../protos/rw/remote_pb')
+const https = require('https')
+const url = require('url')
+
+
+function attachSample(s, samples, metric_type = 'reg'){
+    let sample = new wr.Sample();
+    sample.setTimestampmillis(Math.floor(s.startTimeUnixNano/1000000));
+    if (metric_type == 'reg') {
+        sample.setValue(s.value);
+    }
+    else if (metric_type == 'sum') {
+        sample.setValue(s.sum);
+    }
+    else if (metric_type == 'count') {
+        sample.setValue(s.count);
+    }
+    else if (metric_type == 'avg') {
+        sample.setValue(s.sum/s.count);
+    }
+    samples.push(sample)
+}
+
+function attachLabel(l, labels){
+    let label = new wr.Label();
+    label.setName(l.key);
+    label.setValue(l.value);
+    labels.push(label)
+}
+
+function convertToTsLSamples(metric, ts, metric_type = 'reg'){
```

Comment:
what the metric_type is used for as you dont use it in this function

Reply:
`metric_type = 'reg'` Is the [default value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters?retiredLocale=he) in case we dont specify another type.